instrumented-tests/Integration/IntegrationTests/LoggingIntegrationTests.swift

```diff
@@ -74,7 +77,7 @@ class LoggingIntegrationTests: IntegrationTests {
         logMatchers[5].assertMessage(equals: "critical message")
 
         logMatchers.forEach { matcher in
-            matcher.assertDate(matches: { $0.isNotOlderThan(seconds: 60) })
+            matcher.assertDate(matches: { Date().timeIntervalSince($0) < Constants.logsDeliveryTime * 2 })
```

Comment:
Times 2 because the backend team told you that the reasonable amount of acceptable time is 1 minute maximum?

I was just wondering ;)

Reply:
Integration test uses our [HTTPServerMock](https://github.com/DataDog/dd-sdk-ios/tree/a1bd390ab58b79f6607afb0b6b9bc6553c499fae/instrumented-tests/http-server-mock) (python server running on localhost). The server is started in test's pre-action and terminated in post-action. No real backend involved.

This `60s` is an arbitrary value coming from the fact that after writing the last message, this test waits `30s`, leaving the logs upload worker enough time for delivery:
```swift
logger.critical("critical message", attributes: ["attribute": "value"])

// Wait for delivery
Thread.sleep(forTimeInterval: Constants.logsDeliveryTime)
```
In theory, given negligible latency on localhost, waiting anything over `30s` (like `30.5s`) should be enough, but I just prefer natural numbers, that's why `2 * 30s` 😜.


💡 Moreover, the `30s` of a wait time, comes from the SDK upload heuristic: it first tries the upload after `5s` and if log was found it reduces the interval by the factor of `0.9` OR if no log was found, it bumps it to `20s` for one time.